### PR TITLE
Included desription of ArduinoOTA support and other corrections listed

### PR DESCRIPTION
**Pull request by krzychb**
1. Include ArduinoOTA as the way to go - http://www.esp8266.com/viewtopic.php?p=33231#p33231 and https://github.com/esp8266/Arduino/pull/984#issuecomment-155579984
2. Need to update boards.txt to have OTA for NodeMCU 1.0 in stable or staging platform package
3. Address possible confusion regarding DNS_SD_Arduino_OTA.ino - https://github.com/esp8266/Arduino/issues/1001 and https://github.com/esp8266/Arduino/pull/984#issuecomment-155805311

### CODE REVIEW

File: doc/ota_updates/ota_updates.md [hunk 1]
```diff
@@ -18,13 +18,44 @@ OTA may be done from:
...
+```
+
+The following chapters provide more details and specific methods of doing OTA.
 
 
 ## Basic Requirements
 
 - Flash chip size is 2x the size of the sketch.
 
+Two separate, same size memory locations are used alternately. While existing sketch is executed from one location, the new sketch is saved to the other. After reset the new sketch is executed directly from where saved. Location of old sketch will be used for next OTA upload.
```
Comment (Links2004) [inline doc/ota_updates/ota_updates.md]: the sketch is copied over the old one at startup.

take a look here:
https://github.com/esp8266/Arduino/blob/master/cores/esp8266/Updater.cpp#L151-L156
https://github.com/esp8266/Arduino/tree/master/bootloaders/eboot

**Comment by krzychb** [comment]
Thank you for review Links2004
I would delete this paragraph. 
Could you do it for me or I should submit another pull request?
Krzysztof

**Comment by Links2004** [comment]
I will merge it in

**Comment by Links2004** [comment]
i close this pull request, will merge #1024 with you change as soon the Travis has finished the build checks.

**Pull request closed without merging**